packages/core/src/modules/credentials/protocol/v2/__tests__/v2credentials-architecture.test.ts

```diff
@@ -85,7 +91,11 @@ describe('V2 Credential Architecture', () => {
       const version: CredentialProtocolVersion = CredentialProtocolVersion.V2
       const service: CredentialService = api.getService(version)
       const formatService: CredentialFormatService = service.getFormatService(CredentialFormatType.Indy)
-      const { format: formats, attachment: filtersAttach } = await formatService.createProposal(proposal)
+
+      const { format: formats, attachment: filtersAttach } = await formatService.createProposal(
```

Comment:
I think it is a bit cleaner to just use the same name as the return value.

It also confuses me a bit if the return value is `format` as a signular and we bind it to a variable as `formats`, plural. Which is it?

```suggestion
      const { format, attachment } = await formatService.createProposal(
```